examples/llava/clip.cpp

```diff
@@ -499,19 +498,15 @@ struct clip_ctx * clip_model_load(const char * fname, const int verbosity = 1) {
     }
 
     // data
-    size_t ctx_size = 0;
+    size_t buffer_size = 0;
     {
-        const int n_tensors = gguf_get_n_tensors(ctx);
-
         for (int i = 0; i < n_tensors; ++i) {
             const char * name = gguf_get_tensor_name(ctx, i);
             const size_t offset = gguf_get_tensor_offset(ctx, i);
-
             struct ggml_tensor * cur = ggml_get_tensor(meta, name);
-            ctx_size += sizeof(struct ggml_tensor) + GGML_OBJECT_SIZE;
             size_t tensor_size = ggml_nbytes(cur);
             size_t padded_size = ggml_nbytes_pad(cur);
-            ctx_size += padded_size;
+            buffer_size += padded_size;
```

Comment:
`ggml_nbytes_pad` may not be enough for the CUDA backend, since it uses a higher alignment than the CPU backend and may require additional padding. The correct way to handle this is like `ggml_backend_alloc_ctx_tensors` (not yet merged), but for now adding 128 bytes to the buffer size per tensor should be good enough.